dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -561,7 +566,7 @@ private String getRequirementOrHint(List<Object> requirements, List<Object> hint
      * @param yaml
      * @return
      */
-    private String parseSecondaryFile(String stepDockerRequirement, String secondaryFileContents, Gson gson, Yaml yaml) {
+    private String parseSecondaryFile(String stepDockerRequirement, String secondaryFileContents, Gson gson, Yaml yaml) { //
```

Comment:
Is there any reason for this? Just added two slashes at the end of the line? Just trying to make sure I understand.

Reply:
> Was here, waiting for tests.
> 
> If you're not sure about `$` being optional, should this handle both cases?

More discussion here https://github.com/DockstoreTestUser2/dockstore-whalesay-imports/pull/1

Reply:
Ah no, this was a mistake. Left in when I was debugging